builtin/logical/pki/backend.go

```diff
@@ -261,14 +264,17 @@ func (b *backend) metricsWrap(callType string, roleMode int, ofunc roleOperation
 
 // initialize is used to perform a possible PKI storage migration if needed
 func (b *backend) initialize(ctx context.Context, _ *logical.InitializationRequest) error {
+	// Load up our current pki storage state, no matter the host type we are on.
+	b.updatePkiStorageVersion(ctx)
```

Comment:
I forgot to mention this on the last request, but one thing I noticed during our call today is that `resolveIssuerCRLPath` needs to return `legacyCRLPath, nil` when the cluster isn't migrated -- do you want to handle this as part of this PR or should I handle that in a separate PR once this one merges?

Reply:
Just pushed a commit for this.

Reply:
thank you!